### PR TITLE
remove ref to testCondDBDict which is not part of this package any more

### PR DESCRIPTION
**Pull request by smuzaffar**

### CODE REVIEW

**Comment by smuzaffar** [comment]
please test

**Comment by davidlt** [comment]
I commented on this Saturday. _testRootStreaming_ needs the dictionary, otherwise it will fail. The dictionary files and selections files are kept, but for some reason building dictionary was disabled. (See #11120).

It was removed to silence some warning from SCRAM: https://github.com/apfeiffer1/cmssw/commit/c7b8f3e62d0e1a78de336ddc77f84e6012e3369a#diff-156022378736cc699d979c4a9edf1664

**Comment by smuzaffar** [comment]
@davidlt, ah ok, yes testRootStreaming need the MyTestData dictionary in order to run. I think just added a empty cc file and generating a library in this test directory should allow to generate the missing dictionary

**Comment by davidlt** [comment]
There is all the files need for _genreflex_ command. That generates a C++ file for dictionary, aren't that enough?

**Comment by smuzaffar** [comment]
yes but one needs to explicitly tell scram that it should also generate a library along with other products on the test directory. Does does it automatically for src sub-directory but of all the other sub-dirs one needs to explicitly ask for build products. I have updated the PR where I have just updated the test/BuildFile.xml  so that it can generate a share library (magic part is that here it doe snot need a cc file)

**Comment by smuzaffar** [comment]
please test

File: CondCore/CondDB/test/BuildFile.xml [hunk 1]
```diff
@@ -17,11 +18,9 @@
 </bin>
 
 <bin   file="testRootStreaming.cpp" name="testRootStreaming">
-  <lib   name="testCondDBDict"/>
```
Comment (davidlt) [inline CondCore/CondDB/test/BuildFile.xml]: Does SCRAM automatically link the dictionary to everything, or we need to bring back this statement?

Comment (smuzaffar) [inline CondCore/CondDB/test/BuildFile.xml]: I think root is going to take care of it by reading the testCondDBDict_xr.rootmap file ...right? at least unit tests runs locally without explicitly linking to testCondDBDict

**Comment by ggovi** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**